cpp/examples/tutorial_examples/compute_example.cc

```diff
@@ -0,0 +1,64 @@
+#include <arrow/api.h>
+#include <arrow/result.h>
+#include <arrow/status.h>
+#include <arrow/table.h>
+#include <arrow/compute/api.h>
+
+#include <iostream>
+
+using arrow::Status;
+
+namespace { 
+
+Status RunMain(int argc, char** argv) {
+ 
+  // Saving and Loading Tables
+
+  arrow::Int32Builder int32builder;
+  int32_t some_nums_raw[5] = {34, 624, 2223, 5654, 4356};
+  ARROW_RETURN_NOT_OK(int32builder.AppendValues(some_nums_raw, 5));
+  std::shared_ptr<arrow::Array> some_nums;
+  ARROW_ASSIGN_OR_RAISE(some_nums, int32builder.Finish());
+
+  int32_t more_nums_raw[5] = {75342, 23, 64, 17, 736};
+  ARROW_RETURN_NOT_OK(int32builder.AppendValues(more_nums_raw, 5));
+  std::shared_ptr<arrow::Array> more_nums;
+  ARROW_ASSIGN_OR_RAISE(more_nums, int32builder.Finish());
+
+  std::shared_ptr<arrow::Field> field_a, field_b;
+  std::shared_ptr<arrow::Schema> schema;
+
+  field_a = arrow::field("A", arrow::int32());
+  field_b = arrow::field("B", arrow::int32());
+
+  schema = arrow::schema({field_a, field_b});
+
+  std::shared_ptr<arrow::Table> table; 
+  table = arrow::Table::Make(schema, {some_nums, more_nums}, 5);
+
+  // Performing Computations
```

Comment:
Be explicit on the type of computations performed. I think a bit more detail will make these examples easy to digest to Arrow novices.

Reply:
Working on this and the comment above currently.

Reply:
Added said explicitness.